### DISCUSSION TITLE
Sumatra thumbnails... again

### DISCUSSION THREAD

**Discussion by RusskyVolk** [sumatrapdfreader/sumatrapdf#4329] [Q&A]
Good afternoon. Sumatra v. 3.5.2. 64-bit. Just installed on a new rig running with a Win10 Pro; during install clicked 'options' and enabled previews.

However, no pdf file is showing thumbnails. Tried reinstalling, issue persists. Always had a soft spot for Sumatra, which did everything better than Adobe, but this thumbs issue is a big one when you go through hundreds of pdfs in a sitting.

Please advise.

_Note: My level of understanding: moderately noob. ;) Means, I can follow directions very well, but cannot write code or do anything remotely like that._

**Comment by GitHubRulesOK** [comment]
It should work ! unless there is interference so in my case with Acrobat running need to ensure Acrobat preview is disabled by unchecking
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/591b7fa7-96a7-416f-8f93-28a6a331bf4c) **otherwise there is a conflict**
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/8380d68a-df52-4853-809b-648ea939386e)

now I know that's off I can close Explorer **so no previews are in conflict** and reinstall by overwrite current SumatraPDF with preview option selected again, this will work for days until anytime Adobe triggers reinforcing itself !! Previews cannot work with open password protected or corrupt PDFs

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/987cda19-3d0a-40d2-b8c5-b29047f961f2)


Once previews are working then thumbnails can be seen
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/14258436-4e7d-48d9-a0ab-f6d30956ecb6)

**Comment by RusskyVolk** [comment]
See, I told you - noob. I should have mentioned that I do not have Adobe PDF installed at all. (My comparison was general. I prefer Sumatra exactly because Adobe is no match.)

I do have one thing from Adobe though - an old Photoshop (Photoshop CS). It shouldn't interfere here, should it? Although, when I installed it, I did scroll through file association... How would I check that?

**Reply by GitHubRulesOK** [reply]
Sorry decades since using Photoshop but I suspect it may have had a preview option when it was installed ?

Simplest I can suggest is first uninstall SumatraPDF and reboot to see if the previews are available by any other preview handler
IF so then you need to decide if you stick with that one but  it may not include thumbnails.

AFAIK thumbnails are usually a different feature to **some** previewers (may even be 2 different apps) but in SumatraPDF  they are in effect one combined dll function. 

The key to it working is it must be the one loaded when Windows starts Explorer
For testing differences is best to reboot (or fully log off log on) when looking for functionality differences.
